src/components/LHNOptionsList/index.js

```diff
@@ -0,0 +1,19 @@
+import React, {forwardRef} from 'react';
+import BaseOptionsListLHN from './BaseOptionsListLHN';
+import withWindowDimensions from '../withWindowDimensions';
+import {propTypes, defaultProps} from './optionsListPropTypesLHN';
+
+const OptionsListLHN = forwardRef((props, ref) => (
+    <BaseOptionsListLHN
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        {...props}
+        ref={ref}
+        keyboardDismissMode={props.isSmallScreenWidth ? 'on-drag' : 'none'}
```

Comment:
so, check me on this please, but I'm thinking that this is no longer necessary because this view can't have a keyboard over it. I think the only thing I can think of would be clicking on the search icon to search for a chat, but that opens up an entirely different view.

Reply:
Yep that's exactly what I was thinking.